docs/tutorials/projects/filtered-camera.md

```diff
@@ -181,7 +180,18 @@ Once your dataset is ready, train a new ML model on that dataset.
 
 For more information, see [Train a model](/ml/train-model/).
 
-### Upload an existing ML model
+### Use an existing ML model
+
+You can either [use an existing model from the registry](#select-an-existing-model-from-the-registry) or [upload an ML model](#upload-an-existing-ml-model) trained outside the Viam platform.
+
+#### Select an existing model from the Registry
+
+The Viam registry hosts trained ML models that users have made public, which you can use to deploy classifiers or detectors for your use case onto your robot instead of training your own.
```

Comment:
```suggestion
The Viam registry provides pre-trained ML models that users have made public, which you can use to deploy classifiers or detectors for your use case onto your machine instead of training your own.
```

Nit: `host` suggest compute node, or webserver (active provenance of a service, rather than passive offering of a downloadable artifact). `provides` or `offers` or similar is more familiar for "resource can be found here"

`pre-trained` to match usage elsewhere on this page. Line 163 above walks us through uploading a not-pre-trained trained model, for example.

Reply:
I don't really understand what `pre-trained` is supposed to mean, there is no additional training possible (currently). And any model that gets uploaded is already trained? What does pre-trained mean to you?

Reply:
Gotcha, I see the following two types:
- Pre-trained: Upload a model file & labels file, or add a module that provides an ML model. 
- Train yourself: Upload a bunch of photos, add labels, and train a new model on those photos.

Both are trained models, the former is done before this tutorial starts, the latter as part of this tutorial (so `pre-` helps to order operations in that regard).

The alternative would be to remove `trained` everywhere instead, and just call them `ML models` IMO:
- Upload an ML model, or add a module that provides an ML model.
- Train your own ML model.

But mostly: we use `pre-trained` elsewhere on this page, so should ideally match here as well. Across the docs, however, there are more instances of `trained` over `pre-trained`, so feel free to move this whole page to `trained` if you like, I could see that working 👍